packages/fuelchain/src/parser/parseEvmType.ts

```diff
@@ -0,0 +1,193 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-case-declarations */
+/* eslint-disable default-case */
+/* eslint-disable @typescript-eslint/no-use-before-define */
+/* eslint-disable no-plusplus */
+/* eslint-disable radix */
```

Comment:
if we're ok in getting more and more different from `typechain` code, would be nice to fix this anti-patterns pointed by eslint

Reply:
there's more along other files, should fix there too if possible